docs/UserGuide.md

```diff
@@ -103,17 +103,18 @@ Format: `:wq`
 
 ## Command Summary
 
-| Action              | Format                                   | Examples        |
-|---------------------|------------------------------------------|-----------------|
-| **Add exercise**    | :a <exercise> <weight(kg)> <sets> <reps> | :a squat 60 5 5 |
-| **Delete exercise** | :d <index>                               | :d 3            |
-| **List exercises**  | :ls                                      | :ls             |
-| **Help menu**       | :help                                    | :help           |
-| **Exit program**    | :wq                                      | :wq             |
+| Action              | Format                                           | Examples                |
+|---------------------|--------------------------------------------------|-------------------------|
+| **Add exercise**    | :a n/<exercise> w/<weight(kg)> s/<sets> r/<reps> | :a n/Squat w/60 s/5 r/5 |
+| **Delete exercise** | :d <index>                                       | :d 3                    |
+| **List exercises**  | :ls                                              | :ls                     |
+| **Help menu**       | :help                                            | :help                   |
+| **Exit program**    | :wq                                              | :wq                     |
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Glossary of Terminologies 
-* **Exercise** : Activity requiring physical effort, carried out to sustain or improve health and fitness
+* **Exercise** : Physical activity done in a regular gym that is planned, structured, and repetitive and has as a final 
+or an intermediate objective to the improvement or maintenance of physical fitness
```

Comment:
Seems to be too long; if ultimately our goal is to emphasise that we require weight/sets/reps to be logged, perhaps "structured and repetitive" is enough.
Also debatable if we want to include the objective of exercise in the definition.

Maybe "physical activity, done in a regular gym, that is structured and repetitive, usually/possibly involving some weights (carried out to sustain or improve health and fitness)"

Reply:
Sounds great, will make the necessary changes

Reply:
I don't think the purpose of exercise is necessary, rather keep it concise. 

Perhaps consider. "Physical activity that can be done in groups of successive repetitions, possibly involving some external weights."